docs/pages/kubernetes-access/guides/discovery/aws.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Teleport EKS auto-discovery (Preview)
+description: Auto-discovery of EKS clusters in AWS cloud.
+---
+
+Similarly to [AKS discovery](./azure.mdx), the EKS Discovery can automatically 
+discover any EKS cluster running and enroll it if its tags match the configured labels. 
+
+As mentioned in [Kubernetes Discovery](../discovery.mdx), the discovery process 
```

Comment:
As with an earlier comment, I think we should use "Kubernetes Service" instead of `kubernetes_service` and "Discovery Service" instead of `discovery_service` when we are talking about the service instead of its configuration block.